### PR TITLE
Grammatical error in secret.md

### PR DESCRIPTION
**Pull request by Btakuski**
Fixing a minor grammatical error in secret.md

### CODE REVIEW

**Comment by Btakuski** [comment]
@k8s-ci-robot please rerun checks

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-26203--kubernetes-io-master-staging.netlify.app/docs/concepts/configuration/secret/#tls-secrets

File: content/en/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ ConfigMap.
 
 ### TLS secrets
 
-Kubernetes provides a builtin Secret type `kubernetes.io/tls` for to storing
+Kubernetes provides a builtin Secret type `kubernetes.io/tls` for storing
```
Comment (bradtopol) [inline content/en/docs/concepts/configuration/secret.md:293]: Great catch!!!

**Review by bradtopol — COMMENTED** [review]
Great job

File: content/en/docs/concepts/configuration/secret.md (hunk 1 above)
Comment (Btakuski) [reply content/en/docs/concepts/configuration/secret.md:293]: Thanks Brad! It's small, but I'm glad to have finally contributed to OSS 😄

**Pull request merged**